cobalt/dom/media_source.cc

```diff
@@ -145,6 +148,24 @@ bool IsMediaElementUsingMediaSourceBufferedRangeEnabled(
       .value_or(false);
 }
 
+// If this function returns true, communication between HTMLMediaElement and
+// MediaSource objects will be fully proxied between MediaSourceAttachment.
+// The default value is false.
+bool IsMediaElementUsingMediaSourceAttachmentMethodsEnabled(
+    web::EnvironmentSettings* settings) {
+  return GetMediaSettings(settings)
+      .IsMediaElementUsingMediaSourceAttachmentMethodsEnabled()
+      .value_or(false);
+}
+
+// If this function returns true, experimental support for creating MediaSource
```

Comment:
As MSE-in-Worker can only be enabled with `MediaElement.EnableUsingMediaSourceBufferedRange` and `MediaElement.EnableUsingMediaSourceAttachmentMethods`, should we consider to put a comment here mentioning all of them should be true?